corelib-storage/src/main/java/eu/europeana/corelib/solr/entity/TimespanImpl.java

```diff
@@ -20,11 +20,12 @@
 
 	private Map<String,List<String>> begin;
 	private Map<String,List<String>> end;
-	private Map<String,List<String>> isPartOf;
```

Comment:
the schema.org serialization for Timespans doesn't require any change in this class, please revert back the changes to the ones from the develop version

Reply:
Reverted!